Themes/default/css/index.css

```diff
@@ -57,6 +58,10 @@ textarea, textarea.editor {
 input, button, select {
 	font-size: 0.9em;
 }
+/* Height fixation */
+input, select {
```

Comment:
because they have different heights ? and different browsers natively handle height/width differently. We need a standart on that.